pkg/api/v1/types.go

```diff
@@ -1631,7 +1631,10 @@ type Endpoints struct {
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
 	ObjectMeta `json:"metadata,omitempty"`
 
-	// The set of all endpoints is the union of all subsets.
+	// The set of all endpoints is the union of all subsets. Addresses are placed into
+	// subsets according to the IPs they share. If a single address has multiple ports,
```

Comment:
Multiple Containers

On Sep 15, 2015, at 8:37 PM, Tim Hockin <notifications@github.com> wrote:

In pkg/api/v1/types.go
<https://github.com/kubernetes/kubernetes/pull/13778#discussion_r39583236>:

> @@ -1631,7 +1631,10 @@ type Endpoints struct {
>  	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
>  	ObjectMeta `json:"metadata,omitempty"`
>
> -	// The set of all endpoints is the union of all subsets.
> +	// The set of all endpoints is the union of all subsets. Addresses are placed into
> +	// subsets according to the IPs they share. If a single address has multiple ports,

What does it mean for a port to be ready? AFAIU readiness is a property of
the pod, not of a pod-port?

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/13778/files#r39583236>.